test/util/e2e.go

```diff
@@ -236,8 +236,7 @@ func waitForOperatorAvailability(ctx context.Context, k8sClient client.Client, k
 }
 
 func WaitForKueueAvailability(ctx context.Context, k8sClient client.Client) {
-	kueueNS := GetKueueNamespace()
```

Comment:
won't this return "kueue-system"? Maybe we don't need to change
https://github.com/kubernetes-sigs/kueue/blob/217788b39334b9cb3bbc7bab2cdfaee7860c8ac6/test/util/e2e.go#L66-L71

Reply:
I am not sure what you mean here.